src/cargo/core/compiler/layout.rs

```diff
@@ -164,7 +164,6 @@ impl Layout {
         let lock = dest.open_rw(".cargo-lock", ws.config(), "build directory")?;
         let root = root.into_path_unlocked();
         let dest = dest.into_path_unlocked();
-
```

Comment:
This seems like a spurious change that would be best left out to avoid unnecessary conflicts.

Reply:
Have a commit to fix this (and another superfluous formatting change), I think I'll wait to push it though so I don't restart the GH actions...